templates/serverpod_templates/projectname_server/lib/server.dart

```diff
@@ -3,7 +3,7 @@ import 'package:serverpod/serverpod.dart';
 import 'src/generated/protocol.dart';
 import 'src/generated/endpoints.dart';
 
-// This is the starting point of your Serverpod server.
+/// This is the starting point of your Serverpod server.
```

Comment:
```suggestion
/// The starting point of the Serverpod server.
```

To make it more dartdoccy.